packages/ui-navigation/src/AppNav/props.ts

```diff
@@ -80,7 +80,7 @@ type AppNavOwnProps = {
    * the navigation changes. Passes in the `visibleItemsCount` as
    * a parameter.
    */
-  onUpdate?: ({ visibleItemsCount }: { visibleItemsCount: number }) => void
+  onUpdate?: (visibleItemsCount: { visibleItemsCount: number }) => void
```

Comment:
this was needed because react-docgen was failing with the old code, see https://github.com/reactjs/react-docgen/issues/827

Reply:
This was fixed in react-docgen@6.0.3.